crates/biome_cli/src/execute/migrate.rs

```diff
@@ -173,7 +173,7 @@ pub(crate) fn run(migrate_payload: MigratePayload) -> Result<(), CliDiagnostic>
     } else if configuration_content != new_configuration_content || has_deprecated_configuration {
         if write {
             let mut configuration_file = if has_deprecated_configuration {
-                let biome_file_path = configuration_directory_path.join(fs.config_name());
+                let biome_file_path = configuration_directory_path.join(ConfigName::biome_json());
```

Comment:
In the PR description you say that `migrate` handle both config. I'm only seeing one here. Is this normal?

Reply:
good catch

Reply:
This code path is expected because it migrates a `rome.json` file to a `biome.json` file.